containers/rightToLeft/Provider.js

```diff
@@ -11,7 +11,7 @@ const Provider = ({ children }) => {
         if (isRTL) {
             document.documentElement.dir = 'rtl';
         } else {
-            document.documentElement.dir = undefined;
+            document.documentElement.dir = '';
```

Comment:
I assumed 'undefined' would not add `dir` at all, but otherwise, I think the only possible values are `rtl` or 'ltr`, not empty (https://www.w3.org/TR/REC-html40/struct/dirlang.html)

Does it not add anything to `dir` attribute, or leaves it empty?

Reply:
Turns out it adds an empty attribute 
![Screenshot 2019-08-20 at 14 18 41](https://user-images.githubusercontent.com/352607/63346540-78edd600-c355-11e9-8334-6fbbf4c8fa6f.png)